pkg/controllermanager/apis/config/v1alpha1/types.go

```diff
@@ -37,6 +37,8 @@ type ControllerManagerConfiguration struct {
 	LeaderElection LeaderElectionConfiguration `json:"leaderElection"`
 	// LogLevel is the level/severity for the logs. Must be one of [info,debug,error].
 	LogLevel string `json:"logLevel"`
+	// LogFormat is the output format for the logs. Must be one of [text,json].
+	LogFormat string `json:"logFormat"`
```

Comment:
Should this be an optional field (pointer, `omitempty`), that is properly defaulted like in the gardenlet config API?

Reply:
As discussed, for the sake of consistency I left it the way it is. Defaulting works fine if `logFormat` is omitted.